docs/cugraph/source/nx_cugraph/index.rst

```diff
@@ -1,9 +1,48 @@
-===============================
-nxCugraph as a NetworkX Backend
-===============================
+nx-cugraph
+-----------
 
+nx-cugraph is a `NetworkX backend <https://networkx.org/documentation/stable/reference/utils.html#backends>`_ that provides **GPU acceleration** to many popular NetworkX algorithms.
+
+By simply `installing and enabling nx-cugraph <https://github.com/rapidsai/cugraph/blob/HEAD/python/nx-cugraph/README.md#install>`_, users can see significant speedup on workflows where performance is hindered by the default NetworkX implementation.  With ``nx-cugraph``, users can have GPU-based, large-scale performance **without** changing their familiar and easy-to-use NetworkX code.
+
+.. code-block:: python
+
+    import pandas as pd
+    import networkx as nx
+
+    url = "https://data.rapids.ai/cugraph/datasets/cit-Patents.csv"
+    df = pd.read_csv(url, sep=" ", names=["src", "dst"], dtype="int32")
+    G = nx.from_pandas_edgelist(df, source="src", target="dst")
+
+    %time result = nx.betweenness_centrality(G, k=10)
+
+.. figure:: ../_static/colab.png
+    :width: 200px
+    :target: https://github.com/rapidsai-community/showcase/blob/632d50068f2dec08a591c98f4c41092eab2dfae4/getting_started_tutorials/accelerated_networkx_demo.ipynb/
+
+    Try it on Google Colab!
+
+
++---------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------+
+| **Zero Code Change Acceleration**                                                           | **Third-Party Library Compatible**                                                                                  |
+|                                                                                             |                                                                                                                     |
+| Just ``%load_ext cudf.pandas`` in Jupyter, or pass ``-m cudf.pandas`` on the command line.  | ``nx_cugraph`` is compatible with most third-party libraries that use pandas.                                       |
++---------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------+
+| **Run the same code on CPU or GPU**                                                         | **100% of the NetworkX API**                                                                                        |
```

Comment:
I wonder if we shouldn't say 100% since that could be interpreted as having GPU acceleration on the entire NX API.  I think it's enough to just say "run the same code with or without GPUs"
```suggestion
| **Run the same code with or without GPUs**                                                         |
```